code/game/objects/structures/window.dm

```diff
@@ -411,6 +411,16 @@
 	health = 300
 	reinf = TRUE
 
+
+/obj/structure/window/reinforced/briefing
+	name = "reinforced safety glass"
+	desc = "A very tough looking glass window with a special rod matrice, probably bullet proof."
+	icon_state = "rwindow"
+	basestate = "rwindow"
+	health = 3000
```

Comment:
given how directional should have half the health of a full tile one and requisitions windows have 2k, I think 1k should be enough.